src/main/scala/is/hail/stats/RegressionUtils.scala

```diff
@@ -54,50 +52,169 @@ object RegressionUtils {
 
     val yArray = yForCompleteSamples.map(_.get).toArray
     if (yArray.toSet.size == 1)
-      fatal(s"Constant phenotype: all complete samples have phenotype ${yArray(0)}")
+      fatal(s"Constant phenotype: all complete samples have phenotype ${ yArray(0) }")
     val y = DenseVector(yArray)
 
     val k = covT.size
     val covArray = covForCompleteSamples.flatMap(1.0 +: _.map(_.get)).toArray
     val cov = new DenseMatrix(
-          rows = n,
-          cols = 1 + k,
-          data = covArray,
-          offset = 0,
-          majorStride = 1 + k,
-          isTranspose = true)
+      rows = n,
+      cols = 1 + k,
+      data = covArray,
+      offset = 0,
+      majorStride = 1 + k,
+      isTranspose = true)
 
     (y, cov, completeSamples)
   }
 
-  def buildGtColumn(gts: Iterable[Option[Int]]): Option[DenseMatrix[Double]] = {
-    val (nCalled, gtSum, allHet) = gts.flatten.foldLeft((0, 0, true))((acc, gt) => (acc._1 + 1, acc._2 + gt, acc._3 && (gt == 1) ))
+  // requires bi-allelic
```

Comment:
Delete these.  They aren't helping.  It's obvious from the code and you'll forget to delete them and they won't be visible in the diff context when you start calling nNonRefAlleles or whatever.